source/ai/rl/model/breakout_bbf.py

```
@@ -0,0 +1,212 @@
+        
+        q = v + a - a.mean(-2,keepdim=True)
+        q = F.softmax(q,-1)
+        
+        return q
+    
+    def network_ema(self, rand_network, target_network, alpha=0.5):
+        for param, param_target in zip(rand_network.parameters(), target_network.parameters()):
+            param_target.data = alpha * param_target.data + (1 - alpha) * param.data.clone()
+
+    def hard_reset(self, random_model=None, alpha=0.5):
+        if random_model == None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)